### ISSUE TITLE
APIV3 switch

### ISSUE THREAD

**Issue by thypon** [defunkt/github-gem#84]
Nothing that involves the use of the Authentication Token works anymore for the new users.

https://github.com/blog/1090-github-api-moving-on

Like stated here we need to switch over the v3 API using OAuth methods.

**Comment by drnic** [comment]
Great _sigh_ 

Cheers
Nic

Dr Nic Williams - VP Engineering
Engine Yard
The Leading Platform as a Service
Mobile: +1 415 860 2185
Skype: nicwilliams
Twitter: @drnic

**Comment by rsofaer** [comment]
Yeah, the gem is quite confusing now, because it directs the user to get a token from a page that no longer has a token on it.

**Comment by kohsuke** [comment]
As a starter, I modified the code to send in OAuth header and not token at https://github.com/kohsuke/github-gem/commit/188b22627c248b433dc2b4ea343fbb6c59f7c976

Various code is still making V2 API call, so this is just one step toward the V3 API migration.

**Comment by froschi** [comment]
Am I assuming correctly that "hub" is taking over here?

**Comment by wamatt** [comment]
A lot of places still call this "official". Kinda confusing as this project looks dormant...
